project/TagGen.scala

```diff
@@ -219,6 +233,71 @@ object TagGen {
       HasChildren("wbr")
     )
 
+    def svgTags: List[TagType] =
+      List(
+        NoChildren("animate"),
+        NoChildren("animateColor"),
+        NoChildren("animateMotion"),
+        NoChildren("animateTransform"),
+        NoChildren("circle"),
+        HasChildren("clipPath"),
+        HasChildren("defs"),
+        HasChildren("desc"),
+        NoChildren("ellipse"),
+        NoChildren("feBlend"),
+        NoChildren("feColorMatrix"),
+        HasChildren("feComponentTransfer"),
+        NoChildren("feComposite"),
+        NoChildren("feConvolveMatrix"),
+        HasChildren("feDiffuseLighting"),
+        NoChildren("feDisplacementMap"),
+        NoChildren("feDistantLight"),
+        NoChildren("feFlood"),
+        NoChildren("feFuncA"),
+        NoChildren("feFuncB"),
+        NoChildren("feFuncG"),
+        NoChildren("feFuncR"),
+        NoChildren("feGaussianBlur"),
+        NoChildren("feImage"),
+        HasChildren("feMerge"),
+        NoChildren("feMergeNode"),
+        NoChildren("feMorphology"),
+        NoChildren("feOffset"),
+        NoChildren("fePointLight"),
+        HasChildren("feSpecularLighting"),
+        NoChildren("feSpotLight"),
+        HasChildren("feTile"),
+        NoChildren("feTurbulence"),
+        HasChildren("filter"),
+        HasChildren("foreignObject"),
+        HasChildren("g"),
+        NoChildren("image"),
+        NoChildren("line"),
+        HasChildren("linearGradient"),
+        HasChildren("marker"),
+        HasChildren("mask"),
+        HasChildren("metadata"),
+        NoChildren("mpath"),
+        NoChildren("path"),
+        HasChildren("pattern"),
+        NoChildren("polygon"),
+        NoChildren("polyline"),
+        HasChildren("radialGradient"),
+        NoChildren("rect"),
+        NoChildren("set"),
+        NoChildren("stop"),
+        HasChildren("switch"),
+        HasChildren("symbol"),
+        HasChildren("textTag", "text"),
+        HasChildren("textPath"),
+        HasChildren("tspan"),
+        NoChildren("use"),
+        NoChildren("view")
+      )
+
+    def svgTagList  = TagList(svgTags, "SVG")
+
+    def allTags: List[TagList] = List(svgTagList)
```

Comment:
This seems unnecessary? Only contains the `svgTagList` and is used once?

Reply:
The idea was to have more lists eventually... I don't know what namespaces other than SVG there are but I was expecting some more :D

Reply:
If you get to the end of this journey without adding more, probably best to keep it clean and remove it.

Reply:
Sorry, I just mean the allTags bit. I'm ok with the TagList I think?